utility/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="org.oppia.android.util">
   <uses-sdk android:targetSdkVersion="28" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Why is this needed here? I know it's needed in the main manifest for the app binary, but it's not clear to me why the utility manifest needs it now.

Reply:
![Screenshot from 2021-07-30 16-55-01](https://user-images.githubusercontent.com/63469151/127646545-a0203536-19c7-4a12-850e-1cccb3c40b51.png)
The `connectivityManager` was showing this error so I added permissions to the manifest file.

Reply:
Do tests pass fine without the extra permission added? Android Studio might be looking at the wrong manifest when coming up with this error (it has issues with multi-module projects), and it may not be a real error.

Reply:
Yes, the tests are passing without the permission. I'll remove the permission from the utility manifest file.

Reply:
Removed permission.